README.md

```diff
@@ -20,15 +20,18 @@ Two alternatives to TensorFlow's distribution strategies are available:
 
 ### Install with Pip
 
+Note that in order to support both tensorflow cpu and gpu in a single package, tf-yarn doesn't depend directly on tensorflow.
```

Comment:
Could you please indicate that we recommend TensorFlow >=1.15.0,<=2.2.0 ?  We did not run test with newer versions and it won't work with older versions